django/contrib/sites/management.py

```diff
@@ -2,13 +2,13 @@
 Creates the default Site object.
 """
 
-from django.apps import apps
+from django.apps import apps as global_apps
 from django.conf import settings
 from django.core.management.color import no_style
 from django.db import DEFAULT_DB_ALIAS, connections, router
 
 
-def create_default_site(app_config, verbosity=2, interactive=True, using=DEFAULT_DB_ALIAS, **kwargs):
+def create_default_site(app_config, verbosity=2, interactive=True, using=DEFAULT_DB_ALIAS, apps=global_apps, **kwargs):
```

Comment:
Yes, we're not using the `apps` specific models (`apps.get_models()`) to preform operations here. Plus this signal is only registered for the `sites` apps sender (https://github.com/django/django/blob/e158ec0ba0b43cbe9b509d4ed5587fa3b2ae3cc6/django/contrib/sites/apps.py#L13)